strapi/strapi
# Respect "Size optimization" setting on image resize

### What does it do?

When "Responsive friendly upload" is enabled, the resized images are using default quality options (80) even with "Size optimization" disabled. This PR just checks whether "Size optimization" is enabled or not during the resizing process.

### Why is it needed?

Resized images are compressed even with "Size optimization" disabled.

### How to test it?

1. Enable "Responsive friendly upload" and disable "Size optimization", upload an image and check the quality and file size of the resized images.
2. Pull this PR and do the same test, you'll notice the resized images are larger in file size and of better quality.
3. Disable "Size optimization" and test again. You'll see the resized images are compressed again.
